docs/en/concepts-and-designs/kubesphere-monitoring.md

```diff
@@ -0,0 +1,43 @@
+# KubeSphere Monitoring
+
+## Overview
+
+The KubeSphere monitoring system comprises many components that work together to achieve overall monitoring functionality. KubeSphere defaults to installing the monitoring module. It will deploy  or create a few components within the namespace `kubesphere-monitoring-system`:
+
+- Prometheus Operator: automates tasks for operating Prometheus instances.
+- Prometheus Servers: scrapes metrics and provides monitoring service.
+- Kube-state-metrics: exposes kubernetes resource metrics.
+- Node-exporter: exposes node metrics.
+
+Besides, the monitoring stack includes CRD resources:
```

Comment:
For crd description, we could refer to prometheus official site: https://github.com/coreos/prometheus-operator
Prometheus: Defines a desired Prometheus deployment
ServiceMonitor: Declaratively specifies how groups of services should be monitored.
PrometheusRule: Defines a desired Prometheus rule file